### ISSUE TITLE
manage.py thumbnail_cleanup throws error on Django 1.11

### ISSUE THREAD

**Issue by jorenham** [SmileyChris/easy-thumbnails#473]
When I run `manage.py thumbnail_cleanup` I get

```
Traceback (most recent call last):
  File "/home/joren/Software/pycharm-2017.2.1/helpers/pycharm/django_manage.py", line 43, in <module>
    run_module(manage_file, None, '__main__', True)
  File "/usr/lib/python3.5/runpy.py", line 205, in run_module
    return _run_module_code(code, init_globals, run_name, mod_spec)
  File "/usr/lib/python3.5/runpy.py", line 96, in _run_module_code
    mod_name, mod_spec, pkg_name, script_name)
  File "/usr/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/joren/Workspace/bootbootboot/bootbootboot/manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/home/joren/Workspace/bootbootboot/lib/python3.5/site-packages/django/core/management/__init__.py", line 363, in execute_from_command_line
    utility.execute()
  File "/home/joren/Workspace/bootbootboot/lib/python3.5/site-packages/django/core/management/__init__.py", line 355, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/joren/Workspace/bootbootboot/lib/python3.5/site-packages/django/core/management/base.py", line 275, in run_from_argv
    parser = self.create_parser(argv[0], argv[1])
  File "/home/joren/Workspace/bootbootboot/lib/python3.5/site-packages/django/core/management/base.py", line 249, in create_parser
    self.add_arguments(parser)
  File "/home/joren/Workspace/bootbootboot/lib/python3.5/site-packages/easy_thumbnails/management/commands/thumbnail_cleanup.py", line 160, in add_arguments
    help='The number of days back in time to clean thumbnails for.')
  File "/usr/lib/python3.5/argparse.py", line 1344, in add_argument
    raise ValueError('%r is not callable' % (type_func,))
ValueError: 'int' is not callable
```

- Django 1.11.4
- Python 3.5

**Comment by racitup** [comment]
Not just django 1.11, I get:

> $ ./manage.py help thumbnail_cleanup
> Traceback (most recent call last):
>   File "./manage.py", line 22, in <module>
>     execute_from_command_line(sys.argv)
>   File "/Users/rich/.pyenv/versions/3.5.2/lib/python3.5/site-packages/django/core/management/__init__.py", line 367, in execute_from_command_line
>     utility.execute()
>   File "/Users/rich/.pyenv/versions/3.5.2/lib/python3.5/site-packages/django/core/management/__init__.py", line 351, in execute
>     self.fetch_command(options.args[0]).print_help(self.prog_name, options.args[0])
>   File "/Users/rich/.pyenv/versions/3.5.2/lib/python3.5/site-packages/django/core/management/base.py", line 274, in print_help
>     parser = self.create_parser(prog_name, subcommand)
>   File "/Users/rich/.pyenv/versions/3.5.2/lib/python3.5/site-packages/django/core/management/base.py", line 260, in create_parser
>     self.add_arguments(parser)
>   File "/Users/rich/.pyenv/versions/3.5.2/lib/python3.5/site-packages/easy_thumbnails/management/commands/thumbnail_cleanup.py", line 160, in add_arguments
>     help='The number of days back in time to clean thumbnails for.')
>   File "/Users/rich/.pyenv/versions/3.5.2/lib/python3.5/argparse.py", line 1335, in add_argument
>     raise ValueError('%r is not callable' % (type_func,))
> ValueError: 'int' is not callable

django 1.10.7
python 3.5.2
easy-thumbnails 2.4.1

**Comment by racitup** [comment]
Duplicate of #450

**Comment by racitup** [comment]
The fix is currently in the master branch but not yet released or tagged

**Comment by jrief** [comment]
Just released 2.4.2 on PyPI

**Comment by racitup** [comment]
Thanks! You get everywhere Jacob! 😉
